docs/configuration/vpn.md

```diff
@@ -80,11 +80,13 @@ Set up a ProxyVM as a VPN gateway using NetworkManager
    ip6tables -I FORWARD -i eth0 -j DROP
    ```
 
-6. Configure your AppVMs to use the new VM as a NetVM.
+6. (Optional) Configure the VPN Qube firewall rules so it **only** allows the VPN endpoint address. This is much quicker and can be done graphically, unlike the optional points mentioned above.
```

Comment:
The problem with that approach is that it doesn't guarantee that packets from the AppVMs to the endpoint address are really going through the VPN. An attacker controlling any routing node (eg. a rogue wifi AP at airports) would thus be able to see unencrypted packets from the AppVMs if the VPN endpoint is down (which he/she could easily trigger by dropping/altering packets to the VPN endpoint, which would eventually "break" the connection).
Obviously this is is quite theoritical but still ; maybe add a comment that this approach - while easier to configure - is a less "strict" than the one above ?

Reply:
That's right. There are lots of possible scenarios and individual like me cannot consider. It's a good thing you mentioned these.
Meanwhile, before I'll add this information, we will have a forum discussion [here](https://forum.qubes-os.org/t/bluetooth-and-vpn-guides/14329/2) on how this guide could be enhanced even further - maybe it'll inspire me to add more improvements before this PR gets accepted.

Reply:
Added an appropriate mention on that + a link to your comment. Everything has been explained clearly in there. :smiley: